feat(content): New Alien Faction: The Gegno (#6610)
endless-sky/endless-sky: data/gegno/gegno ships.txt

# Copyright (c) 2022 by Saugia
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A

> # PARTICULAR PURPOSE.  See the GNU General Public License for more details.

Reviewer: ```suggestion
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.
```

	explode "small explosion" 26
	"final explode" "final explosion large"
	description `Over time, the Gegno needed larger vessels to carry materials to and from new worlds - ones that would be easier to build and operate than their older, outdated City-Ships. Conglomerates are built to haul large amounts of cargo and personnel across Gegno space.`

ship Gypsum
	sprite "ship/gegno gypsum"
	thumbnail "thumbnail/gegno gypsum"
	attributes
		category "Light Freighter"
		licenses
			"Gegno Civilian"
		cost 1692000
		hull 5200
		"required crew" 18
		bunks 40
		mass 380
		drag 3
		"heat dissipation" 0.8
		"fuel capacity" 500
		"cargo space" 115
		"outfit space" 240
		"weapon capacity" 42
		"engine capacity" 120
		weapon
			"blast radius" 72
			"shield damage" 320
			"hull damage" 620
			"hit force" 720
	outfits
		"C7 Brawl Cell"
		Excavator
		Hyperdrive
		"R01 Skirmish Battery"
		"RG15 Torch Steering"
		"RG18 Torch Steering"
		"RG18 Torch Thruster"
	engine 0 124
	gun 0 -117 Excavator
	leak leak 60 50
	leak flame 60 80
	explode "large explosion" 14
	explode "medium explosion" 24
	explode "small explosion" 26
	explode "tiny explosion" 10
	"final explode" "final explosion medium"

> 	description `To fuel their increasing need for materials, the Gegno designing a new ship class for the sole purpose of mining. The resulting platform, the Gypsum, is not much more than a flying drill with cargo containers, engines, and a bridge attached behind it, making it cost-efficient and easy to manufacture.`

Reviewer: ```suggestion
	description `To fuel their increasing need for materials, the Gegno designed a new ship class for the sole purpose of mining. The resulting platform, the Gypsum, is not much more than a flying drill with cargo containers, engines, and a bridge attached behind it, making it cost-efficient and easy to manufacture.`
```
Not sure about this fix, but the original definitely wasn't right.

		licenses
			"Gegno Civilian"
			"Vi Evocati"
		cost 1932000
		hull 12375
		"required crew" 36
		bunks 86
		mass 560
		drag 5.9
		"heat dissipation" 0.7
		"fuel capacity" 500
		"cargo space" 140
		"outfit space" 250
		"weapon capacity" 58
		"engine capacity" 106
		"weapon capacity" 58
		weapon
			"blast radius" 71
			"shield damage" 310
			"hull damage" 660
			"hit force" 700
	outfits
		"C7 Brawl Cell"
		Hyperdrive
		"Irate Cannon" 2
		"Irate Cannon Turret"
		"R02 Battlezone Battery"
		"RG18 Torch Steering"
		"RG18 Torch Thruster"
	engine 0 153.5
	gun -16.5 -181.5 "Irate Cannon"
	gun 16.5 -181.5 "Irate Cannon"
	turret 0 -139.5 "Irate Cannon Turret"
	leak leak 60 50
	leak flame 60 80
	explode "large explosion" 10
	explode "medium explosion" 25
	explode "small explosion" 25
	explode "tiny explosion" 10
	"final explode" "final explosion medium"
	description `Based on the Gegno's old Dolomite, the Schist is a mid-range ship used by the Gegno Vi. It can carry a decent amount of cargo, and has enough bunks to carry a whole platoon of Gegno warriors. It is the primary vessel used to transport recruits to the front lines, and veterans back to home territories.` 

> 	description `	Although is it lightly armed compared to other Vi ships, Schists are still mounted a pair of cannons and a turret so it can bring some type of weaponry to a possible engagement.`

Reviewer: ```suggestion
	description `	Although is it lightly armed compared to other Vi ships, Schists are still mounted with a pair of cannons and a turret so it can bring some type of weaponry to a possible engagement.`
```

ship Diorite
	sprite "ship/gegno diorite"
	thumbnail "thumbnail/gegno diorite"
	attributes
		category "Medium Warship"
		licenses
			"Gegno Civilian"
			"Vi Evocati"
			"Vi Centurion"
		cost 6980000
		hull 11600
		"required crew" 41
		bunks 68
		mass 642
		drag 6.3
		"heat dissipation" 0.5
		"fuel capacity" 500
		"cargo space" 40
		"outfit space" 511
		"weapon capacity" 251
		"engine capacity" 124
		"weapon capacity" 251
		weapon
			"blast radius" 90
			"shield damage" 540
			"hull damage" 770
			"hit force" 810
	outfits
		"C17 Warzone Core"
		"Ballistic Cannon" 2
		Hyperdrive
		"Irate Cannon Turret" 2
		"R03 Warforge Battery"
		"RG18 Torch Thruster"
		"RG3 Torch Steering"

	gun -7 -193
	gun 7 -193
	turret -24 -77
	turret 24 -77 
	engine -22.5 158
	engine 0 164
	engine 22.5 158
	leak leak 30 50
	leak flame 50 90
	explode "large explosion" 9
	explode "medium explosion" 15
	explode "small explosion" 20
	explode "tiny explosion" 17
	"final explode" "final explosion medium"

> 	description "Continuing on with the design concept of the Vi's Eclogite, the Diorite is made by a separate shipyard from other larger Vi vessels. Rather than being a platform for large turreted mounts, the Diorite is focused on having powerful forward weaponry in the form of two mounted Ballistic Cannons and a pair of Irate Cannon Turrets to support them. In conjunction with this configuration, the Diorite is much more maneuverable than other Vi ships in its class and can easily keep a beat on other ships its size, sometimes smaller ones."

Reviewer 2: ```suggestion
	description "Continuing on with the design concept of the Vi's Eclogite, the Diorite is made by a separate shipyard from other larger Vi vessels. Rather than being a platform for large turreted mounts, the Diorite is focused on having powerful forward weaponry in the form of two mounted Ballistic Cannons and a pair of Irate Cannon Turrets to support them. In conjunction with this configuration, the Diorite is much more maneuverable than other Vi ships in its class and can easily keep a beat on other ships its size, and sometimes smaller ones as well."
```

ship Gneiss
	plural "Gneiss"
	sprite "ship/gegno gneiss"
	thumbnail "thumbnail/gegno gneiss"
	attributes
		category "Heavy Warship"
		licenses
			"Gegno Civilian"
			"Vi Evocati"
			"Vi Centurion"
			"Vi Lord"
		cost 15690000
		hull 22275
		"required crew" 142
		bunks 162
		mass 1245
		drag 18
		"heat dissipation" 0.4
		"fuel capacity" 500
		"cargo space" 95
		"outfit space" 765
		"weapon capacity" 412
		"engine capacity" 150
		weapon
			"blast radius" 150
			"shield damage" 1150
			"hull damage" 1570
			"hit force" 1200
	outfits
		"Ballistic Cannon Turret" 3
		"C17 Warzone Core"
		Hyperdrive
		"Irate Cannon" 4
		"R04 Crusade Battery"
		"RG3 Torch Steering"
		"RG3 Torch Thruster"
	engine 0 270
	gun -17 -286 "Irate Cannon"
	gun 17 -286 "Irate Cannon"
	gun -23 -270.5 "Irate Cannon"
	gun 23 -270.5 "Irate Cannon"
	turret 0 -97.5 "Ballistic Cannon Turret"
	turret 0 -23 "Ballistic Cannon Turret"
	turret 0 46 "Ballistic Cannon Turret"
	leak leak 50 50
	leak flame 80 140
	explode "huge explosion" 18
	explode "large explosion" 30
	explode "medium explosion" 46
	explode "small explosion" 40
	explode "tiny explosion" 80
	"final explode" "final explosion large"

> 	description `Gneiss Battleships are the Gegno Vi's true sword & shield in war. They are slightly older than the Hornfels and Schists, but carry the heaviest armament, bearing 3 huge mounts dedicated to holding enormous Ballistic Cannons. Gniesses also have extremely strong armor plating, and can take a hefty beating while firing threatening broadsides at the opposing force.`

Reviewer: ```suggestion
	description `Gneiss Battleships are the Gegno Vi's true sword and shield in war. They are slightly older than the Hornfels and Schists, but carry the heaviest armament, bearing 3 huge mounts dedicated to holding enormous Ballistic Cannons. Gniesses also have extremely strong armor plating, and can take a hefty beating while firing threatening broadsides at the opposing force.`
```

# Copyright (c) 2022 by Saugia
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

# Gegno:

ship Shale
	sprite ship/shale
	thumbnail thumbnail/shale

> 	swizzle 0

Author: Alright. Now that I only have greys + a faction's color on the models, I can do that. The swizzle thing was more from two things: One being they previously had two very different colors on top of greys, and another being ships like the Halite and Kyanite having colored external parts such as generators and tubing. Is that fine if that swizzles?

	attributes
		category Transport
		licenses
			"Gegno Civilian"
		cost 265000
		mass 121
		bunks 7
		"cargo space" 36
		drag 1.8
		"engine capacity" 58
		"fuel capacity" 400
		"heat dissipation" 0.8
		"heat protection" 3
		"ion protection" 3
		hull 760
		"outfit space" 113
		"required crew" 1

> 		shields 0
> 		"weapon capacity" 0

Reviewer 3: No need to specify these attributes, for all of the relevant ships.
```suggestion
```

Author: Noted.

ship Dolomite
	sprite ship/dolomite
	thumbnail thumbnail/dolomite
	swizzle 0
	attributes
		category "Light Freighter"
		licenses
			"Gegno Civilian"
		cost 1692000
		mass 570
		bunks 80
		"cargo space" 155
		drag 5.9
		"engine capacity" 81
		"fuel capacity" 500
		"heat dissipation" 0.7
		"heat protection" 3
		"ion protection" 3
		hull 10500
		"outfit space" 150
		"required crew" 34
		shields 0
		"weapon capacity" 0
		weapon
			"blast radius" 72
			"shield damage" 320
			"hull damage" 620
			"hit force" 720
	outfits
		"C3 Scrap Cell"
		Hyperdrive
		"R01 Skirmish Battery"
		"RG15 Torch Steering" 2
		"RG15 Torch Thruster"
	engine -21 155
	engine 21 155
	leak leak 60 50
	leak flame 60 80
	explode "large explosion" 14
	explode "medium explosion" 24
	explode "small explosion" 26
	explode "tiny explosion" 10
	"final explode" "final explosion medium"

> 	description "Almost as large as a human cruiser, Dolomites are the Gegno's most common jack-of-all-trades cargo vessel in service, having been used since the early days of space colonization. They are used as both passenger liners and cargo vessels, and carry decent amounts of both to note. Due to its popularity and continuous usage, both the Gegno Vi and Gegno Scin factions have developed further on the original design for their own purposes."

Reviewer 3: ```suggestion
	description "Almost as large as a human cruiser, Dolomites are the Gegno's most common cargo vessel in service, having been used since the early days of space colonization. They are used as both passenger liners and cargo vessels. Due to its popularity and continuous usage, both the Gegno Vi and Gegno Scin factions have developed upon on the original design for their own purposes."
```

			"hull damage" 2700
			"hit force" 2100
	outfits
		"Ballistic Cannon Turret" 6
		"C17 Warzone Core"
		"C27 Campaign Core"
		"Chloeric Cannon" 2
		Hyperdrive
		"Irate Cannon" 3
		"R04 Crusade Battery"
		"RG3 Torch Steering" 2
		"RG3 Torch Thruster" 4
	engine 94 297
	engine 55.5 373.5
	engine -22.5 373.5
	engine -52 297
	"steering engine" 131 54.5
		angle -60
		zoom 0.5
		left
	"steering engine" 128.5 116
		zoom 0.5
		angle -60
		left
	"steering engine" -107 161.5
		zoom 0.5
		angle 60
		right
	"steering engine" -121 97
		zoom 0.5
		angle 60
		right
	turret 14 -304.5 "Ballistic Cannon Turret"
	turret 13.5 -219 "Ballistic Cannon Turret"
	turret 27.5 -129 "Ballistic Cannon Turret"
	turret 27 134 "Ballistic Cannon Turret"
	turret 17.5 221.5 "Ballistic Cannon Turret"
	turret 18.5 301.5 "Ballistic Cannon Turret"
	gun 57.5 -385 "Chloeric Cannon"
	gun 57.5 -385 "Chloeric Cannon"
	gun -37 -372.5 "Irate Cannon"
	gun -41 -368 "Irate Cannon"
	gun -45 -362 "Irate Cannon"
	leak leak 50 50
	leak flame 80 140
	explode "huge explosion" 36
	explode "large explosion" 40
	explode "medium explosion" 60
	explode "small explosion" 20
	explode "tiny explosion" 60
	"final explode" "final explosion large"

> 	description "The Augen-class is an ancient battleship design proposed by the Vi as a large weapons platform for their Ballistic Cannons. It was built in secret over many years in the huge canyon shipyard present on the foggy world of Enbi, and for many periods of time its construction was put on hold. The recent civil war with the Gegno Scin gave reason to complete the warship, although it is still somewhat of a relic compared to current vessels in the service of the Vi fleet."

Reviewer 3: ```suggestion
	description "The Augen-class is an ancient battleship proposed by the Vi as a large weapons platform for their Ballistic Cannons. It was built in secret over many years in Enbi's canyon shipyard, and for many periods of time its construction was put on hold. The recent civil war with the Gegno Scin gave reason to complete the warship, although it is still somewhat of a relic compared to current vessels in the service of the Vi fleet."
```

		"shields" 0
		"hull" 13300
		"required crew" 18
		"bunks" 66
		"mass" 760
		"drag" 8.9
		"heat dissipation" 0.7
		"fuel capacity" 400
		"cargo space" 415
		"outfit space" 200
		"weapon capacity" 0
		"engine capacity" 136
		weapon
			"blast radius" 124
			"shield damage" 960
			"hull damage" 1160
			"hit force" 1650
	outfits
		"C7 Brawl Cell"
		"R02 Battlezone Battery"
		
		"RG18 Torch Thruster"
		"RG18 Torch Steering"
		"RG15 Torch Steering"
		"Hyperdrive"
	engine -34.5 220
	engine 34.5 220
	leak "leak" 30 40
	leak "flame" 80 20
	explode "huge explosion" 10
	explode "large explosion" 30
	explode "medium explosion" 38
	explode "small explosion" 26
	"final explode" "final explosion large"
	description `Over time, the Gegno needed larger vessels to carry materials to and from new worlds - ones that would be easier to build and operate than their older, outdated City-Ships. Conglomerates are built to haul large amounts of cargo and personnel across Gegno space.`

ship "Gypsum"
	sprite "ship/gegno gypsum"
	thumbnail "thumbnail/gegno gypsum"
	attributes
		category "Light Freighter"
		licenses
			"Gegno Civilian"
		"cost" 1692000
		"shields" 0
		"hull" 5200
		"required crew" 18

> 		"bunks" 40

Author: They have been laid off now. :(

ship "Diorite"
	sprite "ship/gegno diorite"
	thumbnail "thumbnail/gegno diorite"
	attributes
		category "Medium Warship"
		licenses
			"Vi Centurion"
		"cost" 6980000
		"shields" 0
		"hull" 11600
		"required crew" 41
		"bunks" 68
		"mass" 642
		"drag" 6.3
		"heat dissipation" 0.5
		"fuel capacity" 500
		"cargo space" 40
		"outfit space" 548
		"engine capacity" 124
		"weapon capacity" 285
		weapon
			"blast radius" 90
			"shield damage" 540
			"hull damage" 770
			"hit force" 810
	outfits
		"Ballistic Cannon" 2
		"Chloeric Cannon"
		"Irate Cannon Turret" 2
		
		"C17 Warzone Core"
		"R03 Warforge Battery"
		
		"RG18 Torch Thruster"
		"RG3 Torch Steering"
		"Hyperdrive"
	engine -22.5 158
	engine 0 164
	engine 22.5 158
	gun -7 -193 "Ballistic Cannon"
	gun 7 -193 "Ballistic Cannon"
	gun 0 143 "Chloeric Cannon"
		angle 180
	turret -24 -77 "Irate Cannon Turret"
	turret 24 -77 "Irate Cannon Turret"
	leak "leak" 30 50
	leak "flame" 50 90
	explode "large explosion" 9
	explode "medium explosion" 15
	explode "small explosion" 20
	explode "tiny explosion" 17
	"final explode" "final explosion medium"

> 	description "Continuing on with the design concept of the Vi's Eclogite, the Diorite is made by a separate shipyard from other larger Vi vessels. Rather than being a platform for large turreted mounts, the Diorite is focused on having powerful forward weaponry in the form of two mounted Ballistic Cannons and a pair of Irate Cannon Turrets. In conjunction with this configuration, the Diorite is much more maneuverable than other Vi ships in its class and can easily keep a beat on other ships its size, and sometimes smaller ones as well."

Author: Yes, since the shipyard will be the main shipyard for future Gegno tech ships, I guess it should be named. I'm just worried about the whole, should the player see that once Gegno planets are landable, but perhaps these ships aren't even visible unless you are able to buy them, at which point you'd know the shipyard's name(s).

> 	description "Continuing on with the design concept of the Vi's Eclogite, the Diorite is made by a separate shipyard from other larger Vi vessels. Rather than being a platform for large turreted mounts, the Diorite is focused on having powerful forward weaponry in the form of two mounted Ballistic Cannons and a pair of Irate Cannon Turrets. In conjunction with this configuration, the Diorite is much more maneuverable than other Vi ships in its class and can easily keep a beat on other ships its size, and sometimes smaller ones as well."

Author: Or perhaps, I can leave this as-is, and then change it in the future when I get to a mission point when I know the description or shipyards would be mentioned, that way I'm more deep into what I'm doing with the lore and missions at that point.

> 	description "Continuing on with the design concept of the Vi's Eclogite, the Diorite is made by a separate shipyard from other larger Vi vessels. Rather than being a platform for large turreted mounts, the Diorite is focused on having powerful forward weaponry in the form of two mounted Ballistic Cannons and a pair of Irate Cannon Turrets. In conjunction with this configuration, the Diorite is much more maneuverable than other Vi ships in its class and can easily keep a beat on other ships its size, and sometimes smaller ones as well."

Reviewer 4: We can leave the name up in the air for the moment assuming it actually gets a name in the future.

		angle -90
	gun 93 -27.5 "Irate Cannon"
		angle 100
	gun -86.5 -60 "Irate Cannon"
		angle -50
	gun 107 -11.5 "Irate Cannon"
		angle 35
	gun -124.5 103.5 "Irate Cannon"
		angle -90
	gun 105 142 "Irate Cannon"
		angle 95
	gun -101 194.5 "Irate Cannon"
		angle -125
	gun 94.5 211 "Irate Cannon"
		angle 150
	turret 0 21 "Chloeric Cannon Turret"
	turret -69.5 21 "Irate Cannon Turret"
	leak "leak" 30 50
	leak "flame" 30 80
	leak "big leak" 50 30
	explode "huge explosion" 30
	explode "large explosion" 45
	explode "medium explosion" 35
	explode "small explosion" 25
	"final explode" "final explosion large"
	description `Protoliths are descendants of the Gegno's early City-Ships used to colonize nearby systems. They now carry hundreds of Vi both in and out of war, and huge cargo loads to supply depots. They are not typically used as warships but can support enough weaponry to threaten most smaller vessels, or fend off larger ships long enough for reinforcements to arrive.`

# Gegno Scin:

ship "Halite"
	sprite "ship/gegno halite"
	thumbnail "thumbnail/gegno halite"
	attributes
		category "Fighter"
		licenses
			"Scin Hoplologist"
			"Scin Architect"
		"cost" 129000
		"shields" 240

> 		"hull" 780

Author: -200 Social Credit.

	turret 74.5 13 "Acuit Artillery"
	bay "Fighter" -73 -173.5
		over
	bay "Fighter" 73 -173.5
		over
	bay "Fighter" -70 -98
		over
	bay "Fighter" 70 -98
		over
	bay "Fighter" -65.5 120
		angle -180
		over
	bay "Fighter" 65.5 120
		angle -180
		over
	leak "leak" 50 50
	leak "flame" 80 140
	explode "huge explosion" 18
	explode "large explosion" 30
	explode "medium explosion" 46
	explode "small explosion" 50
	explode "tiny explosion" 40
	"final explode" "final explosion large"
	description `The mighty Feldspar is a Scin creation to behold. As a testbed for the new Halite Fighters, Feldspars bring their own small fleets with them and overwhelm their enemy. They also were designed as a semi-experimental platform, built around a brand new multi-engine system and the latest in Gegno plasma technology.`
	description `	Able to keep a decent stance away from a battle while putting down artillery fire, Feldspars are quite a dangerously well-rounded ship for their size. On the downside, they are more susceptible to hull damage like any other Scin warship, especially when alone. As such, they rely on their fighters since they aren't a dedicated warship themselves.`

> ship "Muscovite"

Reviewer 4: While this fits the theme of naming Gegno ships after rocks/minerals, the name for muscovite is based on a well known Earthly location, so it seems weird to use it as a name for an alien ship.

	thumbnail "thumbnail/gegno muscovite"
	attributes
		category "Utility"
		licenses
			"Scin Architect"
			"Scin Hoplologist"
		"cost" 16460000
		"shields" 0
		"hull" 15400
		"required crew" 38
		"bunks" 98
		"mass" 1146
		"drag" 16.1
		"heat dissipation" 0.9
		"fuel capacity" 600
		"cargo space" 430
		"outfit space" 527
		"weapon capacity" 166
		"engine capacity" 182
		weapon
			"blast radius" 216
			"shield damage" 1920
			"hull damage" 1630
			"hit force" 3200
	outfits
		"Astuit Battery" 2
		"Acuit Artillery"
		
		"C7 Brawl Cell"
		"C17 Warzone Core"
		"R02 Battlezone Battery"
		"R03 Warforge Battery"
		
		"SC-15 Plasma Thrusters"
		"SC-14 Plasma Steering" 2
		"Hyperdrive"
	engine 64 188
	engine -64 188
	"reverse engine" 66 -155
	"reverse engine" -66 -155
	turret 0 -144.5 "Astuit Battery"
	turret 0 -27 "Astuit Battery"
	turret 0 65 "Acuit Artillery"
	bay "Fighter" 0 122
		angle -180
		under
	bay "Fighter" 0 180
		angle -180
		under
	leak "leak" 10 40
	leak "flame" 40 70
	leak "big leak" 30 50
	explode "huge explosion" 15
	explode "large explosion" 42
	explode "medium explosion" 42
	explode "small explosion" 38
	"final explode" "final explosion large"

> 	description `Muscovites are large independent research platforms with adequately-sized cargo holds, allowing them to perform as a reliable heavy freighter for the Gegno Scin. A large number of bunks either help transport groups of Scin scientists to their project destinations, where they perform research or experiments, or allow for mobile on-board demonstrations and observations. Each Muscovite is equipped with a rear fighter docking bay capable of carrying two Halites, typically used to ship Halite fighters around Scin space to resupply carriers. Along with their three dorsal turrets, Muscovites can also be used as a secondary warship when needed.`

Reviewer 4: ```suggestion
	description `Muscovites are large independent research platforms with adequately-sized cargo holds, allowing them to perform as a reliable heavy freighter for the Gegno Scin. A large number of bunks either help transport groups of Scin scientists to their project destinations, or allow for mobile on-board demonstrations and observations. Each Muscovite is equipped with a rear fighter docking bay capable of carrying two Halites, typically used to ship Halite fighters around Scin space to resupply carriers. Along with their three dorsal turrets, Muscovites can also be used as a secondary warship when needed.`
```

> 	description `	Like other Gegno Scin ships, Muscovites themselves are an experimental test bed; they are built around the large engine system tested on Feldspars, making them notably fast for their size.`

Reviewer 4: ```suggestion
	description `	Like other Gegno Scin ships, Muscovites themselves are an experimental test bed; they are built around the same large engine system tested on Feldspars, making them notably fast for their size.`
```

	thumbnail "thumbnail/gegno gypsum"
	attributes
		category "Light Freighter"
		licenses
			"Gegno Civilian"
		"cost" 1692000
		"shields" 0
		"hull" 6000
		"required crew" 5
		"bunks" 15
		"mass" 380
		"drag" 3
		"heat dissipation" 0.8
		"fuel capacity" 500
		"cargo space" 220
		"outfit space" 240
		"weapon capacity" 42
		"engine capacity" 120
		"drill port" 1
		"drill lock" -1
		weapon
			"blast radius" 72
			"shield damage" 320
			"hull damage" 620
			"hit force" 720
	outfits
		"Excavator"
		
		"C7 Brawl Cell"
		"R01 Skirmish Battery"
		
		"RG18 Torch Thruster"
		"RG15 Torch Steering"
		"RG18 Torch Steering"
		"Hyperdrive"
	engine 0 124
	gun 0 -117 Excavator
	leak "leak" 60 50
	leak "flame" 60 80
	explode "large explosion" 14
	explode "medium explosion" 24
	explode "small explosion" 26
	explode "tiny explosion" 10
	"final explode" "final explosion medium"
	description `To fuel their increasing need for materials, the Gegno designed a new ship class for the sole purpose of mining. The resulting platform, the Gypsum, is not much more than a flying drill with cargo containers, engines, and a bridge attached behind it, making it cost-efficient and easy to manufacture.`

> 	description `	Because of the Gegno's limited technology, the Gypsum is much more sluggish than similarly sized ships elsewhere in the galaxy, making it too slow to chase faster asteroids. For those asteroids it can catch up to, the powerful Excavator Drill is able to grind materials out with ease.`

Author: @Zitchas It's inferior because it *can't* go after fast asteroids in-game.

> 	description `	Because of the Gegno's limited technology, the Gypsum is much more sluggish than similarly sized ships elsewhere in the galaxy, making it too slow to chase faster asteroids. For those asteroids it can catch up to, the powerful Excavator Drill is able to grind materials out with ease.`

Reviewer 4: ```suggestion
	description `	Because Gegno engines are not very powerful, the Gypsum has difficulty chasing faster moving asteroids. In order to alleviate this problem without needing to massively increase the engine space on the ship, the Gegno have instead developed the powerful Excavator Drill, which can shreds asteroids in a matter of seconds, meaning the Gypsum only needs to get near an asteroid for a mere moment in order to mine it.`
```

			"blast radius" 450
			"shield damage" 2000
			"hull damage" 2700
			"hit force" 2100
	outfits
		"Ballistic Cannon" 6
		"C17 Warzone Core"
		"C27 Campaign Core"
		"Chloeric Cannon" 2
		Hyperdrive
		"Irate Cannon" 3
		"R04 Crusade Battery"
		"RG3 Torch Steering" 2
		"RG3 Torch Thruster" 4
	engine 94 297
	engine 55.5 373.5
	engine -22.5 373.5
	engine -52 297
	"steering engine" 131 54.5
		angle -60
		zoom 0.5
		left
	"steering engine" 128.5 116
		zoom 0.5
		angle -60
		left
	"steering engine" -107 161.5
		zoom 0.5
		angle 60
		right
	"steering engine" -121 97
		zoom 0.5
		angle 60
		right
	turret 14 -304.5 "Ballistic Cannon"
	turret 13.5 -219 "Ballistic Cannon"
	turret 27.5 -129 "Ballistic Cannon"
	turret 27 134 "Ballistic Cannon"
	turret 17.5 221.5 "Ballistic Cannon"
	turret 18.5 301.5 "Ballistic Cannon"
	gun 57.5 -385 "Chloeric Cannon"
	gun 57.5 -385 "Chloeric Cannon"
	gun -37 -372.5 "Irate Cannon"
	gun -41 -368 "Irate Cannon"
	gun -45 -362 "Irate Cannon"
	leak leak 50 50
	leak flame 80 140
	explode "huge explosion" 36
	explode "large explosion" 40
	explode "medium explosion" 60
	explode "small explosion" 20
	explode "tiny explosion" 60
	"final explode" "final explosion large"

> 	description "The Augen-class is an ancient battleship design proposed by the Vi as a large weapons platform for their Ballistic Cannons. It was built in secret over many years in the huge canyon shipyard present on the foggy world of Enbi, and for many periods of time its construction was put on hold. The recent civil war with the Gegno Scin gave reason to complete the warship, although it is still somewhat of a relic compared to current vessels in the service of the Vi fleet."

Author: Yes, since there is a sister ship. All other ships are their own bases, but the sister ship in lore looks different than the first one built.
